cleansio/censor/censor_realtime_mac.py

```diff
@@ -0,0 +1,191 @@
+""" Censors audio chunks in a continuous stream """
+
+import os
+import threading
+import sounddevice as sd
+import soundfile as sf
+from pydub import AudioSegment
+from audio import improve_accuracy, convert_and_write_chunk, \
+    read_and_convert_audio
+from utils import create_env_var, create_temp_dir, append_before_ext, \
+    time_filename, MacUtil, CHUNK_LEN
+from .censor import Censor
+
+class CensorRealtimeMac(Censor):
+    """ Removes explicits from audio stream in real-time """
+
+    running = True
+
+    def __init__(self, args, explicits):
+        print('Initialzed realtime censor object')
+        super().__init__(explicits, args.output_encoding, args.output_location)
+        self.__switch_audio_source()
+        create_env_var('CLEANSIO_CHUNKS_LIST', '[]')
+        self.args = args
+        self.directory = create_temp_dir()
+        self.chunk_prefix = self.directory + time_filename() + '-'
+        self.clean_file = AudioSegment.empty()
+        self.processing_queue = []
+        self.processing_lock = threading.Lock()
+        self.playback_queue = []
+        self.playback_lock = threading.Lock()
+        self.samplerate = 44100 # Hertz
+        self.duration = 5 # seconds
+
+    def censor(self):
+        """ Censors audio chunks in a continuous stream """
+
+        # Start thread that will analyze and censor recorded chunks
+        processing_thread = threading.Thread(target=self.run)
+        processing_thread.daemon = True
+        processing_thread.start()
+
+        try:
+            # Capture stream from Soundflower, play to speakers
+            with sd.Stream(device=(2, 1),
```

Comment:
The methods for looking for the desired sound device are already in the repo. Please fix the hard code